Source/BSGOutOfMemoryWatchdog.m

```diff
@@ -221,7 +243,24 @@ - (NSMutableDictionary *)generateCacheInfoWithConfig:(BugsnagConfiguration *)con
     app[@"releaseStage"] = config.releaseStage;
     app[@"version"] = systemInfo[@BSG_KSSystemField_BundleShortVersion] ?: @"";
     app[@"bundleVersion"] = systemInfo[@BSG_KSSystemField_BundleVersion] ?: @"";
+#if BSGOOMAvailable
+    UIApplicationState state = [self currentAppState];
+    // The app is in the foreground if the current state is "active" or
+    // "inactive". From the UIApplicationState docs:
+    // > UIApplicationStateActive
+    // >   The app is running in the foreground and currently receiving events.
+    // > UIApplicationStateInactive
+    // >   The app is running in the foreground but is not receiving events.
+    // >   This might happen as a result of an interruption or because the app
+    // >   is transitioning to or from the background.
+    // > UIApplicationStateBackground
+    // >   The app is running in the background.
+    app[@"inForeground"] = @(state == UIApplicationStateInactive
```

Comment:
Is `app.inForeground` reported differently for OOM events vs regular error reports? It'd be good to try and address any discrepancy in how the value is reported if feasible, as otherwise this could lead to confusion.

Reply:
Good idea. Raised a ticket to check if other events are reporting foreground state correctly at launch. This one specifically is isolated since it manages its own state.